charts/stable/openspeedtest/values.yaml

```
@@ -0,0 +1,30 @@
+image:
+  repository: tccr.io/truecharts/openspeedtest
+  pullPolicy: IfNotPresent
+  tag: v0.20.1216@sha256:b18c87da9d022c81edcec0cfa21d32669eb99a5e0418a520569ad70e3de939d8
+
+securityContext:
+  readOnlyRootFilesystem: false
+
+service:
+  main:
+    protocol: HTTP
+    ports:
+      main:
+        targetPort: 3000
+        port: 3000
```

```suggestion
  main:
    protocol: HTTP
    ports:
      main:
        targetPort: 3000
        port: 10250
```
We are using different port for every app in our catalog to avoid port conflicts when users install them with default values.

Currently the port list is not up to date, but this is the next free port